src/plugin.js

```diff
@@ -138,6 +138,18 @@ module.exports = class PluginXrpEscrow extends EventEmitter2 {
     return fulfillment
   }
 
+  async submit (signed) {
```

Comment:
This should be an `_`'d method like `_handleTransaction`, and should go next to `_handleTransaction` so that it's easier to see where `_submitted` is actually being used